res/database/Default/user/stukov_ads.xml

```diff
@@ -120,6 +131,7 @@
 			<description>
 				<de>Kapitalia Lebensversicherungen - Wir verwalten ihr Geld schnell und verlässlich.</de>
 				<en>Kapitalia Life Insurance - We manage your money quickly and reliably.</en>
+				<pl>Kapitalia Life Insurance - Zarządzamy Twoimi pieniędzmi szybko i niezawodnie.</pl>
```

Comment:
Translated. What puzzles me is that the insurance company manages the money?

Reply:
Yes ... you pay your monthly rate which they use for other things - to "increase it" (create value/income with it). So they act a bit like a bank but instead of paying you monthly interests (you know ... theye 3-4% per anno or what is current now) they pay you the interest in case of ... death. The value is of course a bit higher than the normal interest as it is surely somehow based on a compound interest formula.
Anyways ... you give money, they "invest it" (in the hope of increasing it) ... and then they take away their profit (as usual) and the "left overs" are what they give you in case of ... no longer requiring the insurance :)